hack/scripts/update-helm-role.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
+
+#  2025 NVIDIA CORPORATION & AFFILIATES
+#
+#  Licensed under the Apache License, Version 2.0 (the License);
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an AS IS BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+set -o nounset
+set -o pipefail
+set -o errexit
+
+if [[ "${TRACE-0}" == "1" ]]; then
+    set -o xtrace
+fi
+
+INPUT_FILE="config/rbac/role.yaml"
+OUTPUT_FILE="deployment/network-operator/templates/role.yaml"
+
+HEADER='{{/*
+  Copyright 2025 NVIDIA
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+
+  // Code generated by controller-gen. DO NOT EDIT.
```

Comment:
Do we want to reuse the hack/boilerplate.go.txt? I see it wasn't updated in 4 years but it might be good to start using it as a source of copyrights (OSRB process requires us to have correct copyrights in all nvidia files)

Reply:
Done